### ISSUE TITLE
Generalize for hermitian PD (complex)

### ISSUE THREAD

**Issue by chriscoey** [JuliaStats/PDMats.jl#93]
Is this just trivial to do?

**Comment by andreasnoack** [comment]
Yes and no. I think much of it should be as simple as loosening the signatures but there might be places where the package makes use of the elements being real in non-obvious ways so I suspect you'd have to extend the testing a lot to be sure that this actually works.

**Comment by chriscoey** [comment]
Makes sense, thanks!
